platforms/common/src/CommonKernels.cpp

```diff
@@ -3011,7 +3048,8 @@ void CommonCalcCustomGBForceKernel::initialize(const System& system, const Custo
         pairValueDefines["NUM_ATOMS"] = cc.intToString(cc.getNumAtoms());
         pairValueDefines["PADDED_NUM_ATOMS"] = cc.intToString(cc.getPaddedNumAtoms());
         pairValueDefines["NUM_BLOCKS"] = cc.intToString(numAtomBlocks);
-        pairValueDefines["TILE_SIZE"] = "32";
+        pairValueDefines["TILE_SIZE"] = cc.intToString(tileSize);
+        pairValueDefines["tileflags"] = (tileSize > 32 ? "unsigned long" : "unsigned int");
```

Comment:
That should be `mm_ulong` rather than `unsigned long`.  `long` is 32 bits in CUDA, 64 bits in OpenCL.

Reply:
And likewise in similar code below.